pkg/config/config.go

```diff
@@ -964,6 +964,11 @@ func (c *RootConfig) Validate(onExecution bool) error {
 		if err := os.MkdirAll(c.LogDir, 0o700); err != nil {
 			return fmt.Errorf("invalid log_dir: %w", err)
 		}
+		if c.Storage == "devicemapper" {
```

Comment:
I am guessing this is not defined anywhere in c/storage, right?

Reply:
No it's not (unlike for other drivers). More to say, sometimes documentation says the name is `devmapper` which is obviously wrong, but I'm not going to fix this since we're removing devicemapper anyway.